client/src/constants/karmaGap.ts

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+// In Karma GAP our epochs are named "Programs".
+export const PROGRAMS_IDS_TO_EPOCH_NUMBER_MAPPING = {
+  1: '544',
```

Comment:
If it is a simple iteration (+1), we can declare only `544` as a program ID of the first epoch and add number (epoch -1) to get the current epoch program id.

Reply:
It's this way because they created all these epochs one after the other. When E6 starts we can't be sure what id there will be. I'd leave it for now.